src/app/login/login.component.ts

```diff
@@ -22,4 +27,9 @@ export class LoginComponent implements OnInit {
   goToProblem() {
     this.router.navigate(['home/problem'], { relativeTo: this.route });
   }
+
+  onButtonClick($event) {
+    console.log($event);
```

Comment:
Remove `console.log`s or add a TODO to remove it.

Reply:
done